srml/support/src/storage/mod.rs

```diff
@@ -330,6 +330,28 @@ impl<K: Codec, V: Codec, U> StorageMap<K, V> for U where U: hashed::generator::S
 	}
 }
 
+/// A storage map that can be enumerated.
```

Comment:
```suggestion
/// A storage map with values that can be appended to.
```

The rest of the comment is not correct.